application/models/file.go

```diff
@@ -92,6 +99,17 @@ func (f *File) Store(name string, content []byte) *FileUploadError {
 		return &e
 	}
 
+	// if possible, re-encode the image to strip EXIF metadata
+	img, _, err := image.Decode(bytes.NewReader(content))
+	if err == nil {
+		buf := new(bytes.Buffer)
+		if err := jpeg.Encode(buf, img, nil); err == nil {
```

Comment:
Do I understand this correct that we'll decode gif, jpg, png, and webp, and then just encode them all to jpg, essentially converting some file types? That seems fine to me, just making sure I understand.

Reply:
Yes, that's correct. It was a lazy approach to not bother re-encoding back to the original format. Also, there is no webp encoder in the standard library.